plugins/autosuspend.koplugin/main.lua

```diff
@@ -46,7 +41,7 @@ function AutoSuspend:_enabledStandby()
 end
 
 function AutoSuspend:_enabled()
-    return self.auto_suspend_timeout_seconds > 0
+    return Device:canSuspend() and self.auto_suspend_timeout_seconds > 0
```

Comment:
Maybe a stupid remark: `self:_enabled()` is called quite often (e.g. `_schedule()`). Would it make sense to add a property (for example `AutoSuspend.can_suspend`) instead of all calling the method `canSuspend()`?

Reply:
Performance-wise that makes no difference worth mentioning and we generally consider using a method better practice. But also I'm sure both are fine. :-P